packages/aws-cdk-lib/aws-ecs/lib/cluster.ts

```diff
@@ -1254,6 +1319,22 @@ export interface AsgCapacityProviderProps extends AddAutoScalingGroupCapacityOpt
   readonly instanceWarmupPeriod?: number;
 }
 
+/**
+ * Kms Keys for encryption ECS managed storage
+ */
+export interface ManagedStorageConfiguration {
```

Comment:
Can you add and manage the `kmsKey` property to complete the interface declaration ([docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-cluster-managedstorageconfiguration.html))?
The title should then be changed to something like `feat(ecs): support cluster managed storage configuration`

Reply:
yes, however I can't test this since I believe for this I'd need to spin up a EC2 + ECS, don't I?

Reply:
aren't EBS encrypted through their [own settings](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_ec2.Volume.html)? 

I was assuming through exclusion that this is only relevant for *Docker volumes* 
EBS, EFS, FSx encrypted through their own properties

Bind Mounts -> ephemeral, therefore ephemeral stoage config

Docker Volumes -> persistent, ec2, no own encryption setting.

Reply:
Thanks for clarifying! Not sure what that `kmsKey` is used for then 🤔 
Let's leave it like this for now and see what the core team thinks about it.